inspirehep/modules/workflows/tasks/matching.py

```diff
@@ -127,11 +127,23 @@ def fuzzy_match(obj, eng):
 
     fuzzy_match_config = current_app.config['FUZZY_MATCH']
     matches = dedupe_list(match(obj.data, fuzzy_match_config))
-    record_ids = [el['_source']['control_number'] for el in matches]
+    record_ids = [_get_literature_brief(el['_source']) for el in matches]
     obj.extra_data.setdefault('matches', {})['fuzzy'] = record_ids[0:5]
     return bool(record_ids)
 
 
+def _get_literature_brief(literature):
```

Comment:
I think maybe just `literature` is not a specific enough parameter name?

Reply:
what do you suggest? sth more explicit like `literature_record`?